docs/extending/code-overview/object-classes.rst

```diff
@@ -35,13 +35,55 @@ called and ``ExpressionList``, where each list item is either itself
 an ``ExpressionList`` or an object in a class derived from ``Atom``.
 
 The ``Atom`` class we encountered earlier when describing the nodes
-that get created intially from a parse. However there are a few other
-kinds of Atoms or fundamention objects that can appear in an
-Evaluation list. These are
+that get created intially from a parse. Those were:
+
+* ``Number``
+* ``String``
+* ``Symbol``
+* ``Filename``
+
+
+However there are a few other kinds of Atoms or fundamental objects
+that can appear in an Evaluation list.  In the evaluation process,
+other kinds of Atoms can get created. These include things like:
 
 * ``CompiledCode``
+* ``DateObject``
```

Comment:
@mmatera Does `AtomQ[DateObject[{2020, 4, 15}]` evaluate to `True` or `False`?

Reply:
False

Reply:
Yes, that is gone in 80641b7617ce8a38aa40c518ab51c41634ef9b2c and `ByteArray` and  `Dispatch` have been added.